src/main/java/com/faforever/client/game/PlayerCardController.java

```diff
@@ -98,10 +105,13 @@ public void initialize() {
             .when(showing));
     avatarImageView.imageProperty()
         .bind(player.flatMap(PlayerBean::avatarProperty).map(avatarService::loadAvatar).when(showing));
+    divisionImageView.imageProperty()
+            .bind(division.flatMap(SubdivisionBean::smallImageUrlProperty).map(leaderboardService::loadDivisionImage).when(showing));
     playerInfo.textProperty()
         .bind(player.flatMap(PlayerBean::usernameProperty)
-            .flatMap(username -> rating.map(value -> i18n.get("userInfo.tooltipFormat.withRating", username, value))
-                .orElse(i18n.get("userInfo.tooltipFormat.noRating", username)))
+            .map(username -> (rating.get() != null && division.get() == null)
+                ? i18n.get("userInfo.tooltipFormat.withRating", username, rating.get())
+                : i18n.get("userInfo.tooltipFormat.noRating", username))
```

Comment:
The rating would change if we changed the player value of the card.

And if the intention is to hide the rating when we have the division then we should modify the visible property in that case and not the actual value.

Reply:
but there is no visibility property to modify. The formating is done through using a different i18n key

Reply:
Then I would probably change it to a rating label and a division label to make it clearer rather than trying to use the same thing for both

Reply:
I would also generally say that we shouldn't be returning null from methods like getPlayerRating just because the division is set. The display of the values should be happening at a higher level